frontend/Dockerfile

```
@@ -1,61 +1,52 @@
+
+# Commande par défaut pour le serveur de développement
+# La commande réelle sera remplacée par celle du docker-compose.yml (pnpm dev --host)
+CMD ["pnpm", "dev", "--host"]
+# Utiliser une version LTS spécifique de Node.js (branche 22.x), variante slim recommandée pour le développement
+FROM node:22.15.1-slim
+
+WORKDIR /app
+
+# Installer et configurer Corepack pour gérer PNPM
+RUN npm install -g corepack@latest
+RUN corepack enable
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)